src/app/screens/Home/styles.module.scss

```diff
@@ -1,16 +1,103 @@
 @import 'scss/variables/_sizes.scss';
 @import '_colors.scss';
 
-.container {
+$calculator-color: #b2e2d4;
+$display-box-color: #6abeb3;
+
+.calculatorContainer {
   align-items: center;
+  background-color: $calculator-color;
+  border-radius: 10%;
   display: flex;
   flex-direction: column;
   height: calc(100% - #{$topbar-heigth});
   justify-content: flex-start;
+  margin: 20px auto;
+  width: 600px;
+}
+
+.calculatorButton {
+  background-color: $calculator-color !important;
+  border: 3px solid white;
+  border-radius: 100px;
+  color: rgb(97, 91, 91) !important;
+  font-size: 40px !important;
+  height: 100px;
+  width: 100px;
+}
+
+.calculatorButtons {
+  display: grid;
+  grid-template-columns: repeat(5, 1fr);
+  grid-gap: 10px;
+  margin: 5px;
   padding: 40px 20px;
-  width: 100%;
+  width: 98%;
+}
+
+.calculatorDisplayBox {
+  background-color: $display-box-color;
+  border: 3px solid rgba(224, 206, 206, 0.5);
+  border-radius: 15% 15% 0% 0%;
+  display: flex;
+  flex-direction: column;
+  height: calc(100% - #{$topbar-heigth});
+  justify-content: flex-start;
+  margin: 15px;
+  padding: 40px 20px;
+  width: 98%;
+}
+
+.calculatorOperationButton {
+  background-color: rgb(153, 143, 143) !important;
+  color:$calculator-color !important;
+}
+
+.calculatorOnButton{
+  background-color: rgb(129, 201, 185) !important;
+  font-size: 25px !important;
```

Comment:
are all this important really necessary?

Reply:
yes